### DISCUSSION TITLE
How to decrypt a encryped Manifest ID

### DISCUSSION THREAD

**Discussion by ghost** [SteamRE/SteamKit#1193] [General]
Hi there,

I don't know how to do this, The branch that I want to decrypt is password protected. If not having a password is a dealbreaker how could I get it?
This is the encrypted Manifest ID: 9EBFDA1E28C7562230E79390C964945B
Here is a link to the game with the branch I want to decrypt.
https://steamdb.info/app/1128920/depots/?branch=dev_branch

DVT

**Comment by psychonic** [comment]
The password can only be viewed (or given out) by the publisher. If you don't have it, there is no way to get it other than from someone else that has it.
